src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/JudgeDraftOrderAboutToSubmitHandlerTest.java

```diff
@@ -0,0 +1,163 @@
+import uk.gov.hmcts.reform.finrem.caseorchestration.mapper.FinremCaseDetailsMapper;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.EventType;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CaseDocument;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CaseType;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.DocumentCollection;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.DraftDirectionOrder;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.DraftDirectionOrderCollection;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.FinremCaseData;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.FinremCaseDetails;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.State;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.wrapper.DraftDirectionWrapper;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.ContestedOrderApprovedLetterService;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.GenericDocumentService;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.HearingOrderService;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.documentcatergory.UploadedDraftOrderCategoriser;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.verifyNoInteractions;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.TestConstants.AUTH_TOKEN;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.TestSetUpUtils.caseDocument;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.test.Assertions.assertCanHandle;
+
+@ExtendWith(MockitoExtension.class)
+class JudgeDraftOrderAboutToSubmitHandlerTest extends BaseHandlerTestSetup {
+
+    private static final String FILE_URL = "http://dm:80/documents/kbjh87y8y9JHVKKKJVJ";
+    private static final String FILE_BINARY_URL = "http://dm:80/documents/kbjh87y8y9JHVKKKJVJ/binary";
+    private static final String FILE_NAME = "abc.pdf";
+
+    @InjectMocks
+    private JudgeDraftOrderAboutToSubmitHandler handler;
+    @Mock
+    private HearingOrderService hearingOrderService;
+    @Mock
+    private GenericDocumentService genericDocumentService;
+    @Mock
+    private ContestedOrderApprovedLetterService contestedOrderApprovedLetterService;
+
+    @Mock
+    private UploadedDraftOrderCategoriser uploadedDraftOrderCategoriser;
+    @Spy
+    private ObjectMapper objectMapper;
+
+    @BeforeEach
+    void setUp() {
+        FinremCaseDetailsMapper finremCaseDetailsMapper = new FinremCaseDetailsMapper(objectMapper);
+        handler = new JudgeDraftOrderAboutToSubmitHandler(
```

Comment:
If you're manually creating the handler then you don't need mockito to create it - you should remove the @InjectMocks annotation